src/Sulu/Component/Content/Types/SingleIconSelect.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+/*
+ * This file is part of Sulu.
+ *
+ * (c) Sulu GmbH
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Sulu\Component\Content\Types;
+
+use Sulu\Component\Content\SimpleContentType;
+
+class SingleIconSelect extends SimpleContentType
```

Comment:
this will only pass the icon id to the website at the moment. shouldnt it pass the icon set too? 🤔

Reply:
I would not, specially headless setup it is part of the frontend team to decide how the embed the icon font into there build process.